ca/certificates.go

```diff
@@ -444,7 +444,10 @@ func NewRootCA(rootCertBytes, signCertBytes, signKeyBytes []byte, certExpiry tim
 		pool.AddCert(cert)
 	}
 
-	// Calculate the digest for our Root CA bundle
+	// Make sure the root certs are normalized before getting the digest, so that we can generate a consistent digest
+	// even if the root certs are.  Also store the normalized root cert in the RootCA object.
+	rootCertBytes = NormalizePEMs(rootCertBytes)
```

Comment:
Won't this affect the root CA cert we return to client that use the `GetRootCACertificate` RPC? If the certificate was originally unnormalized and we start returning a normalized version, it would no longer match the certificate hash in the join token.